src/control/menu/MenuItem.tsx

```diff
@@ -9,27 +9,41 @@ const StyledBox = styled(Box)`
     transition: background-color .2s ease-in-out, border-color .2s ease-in-out;
 `;
 
-type InteractionState = "normal" | "active" | "disabled";
+type InteractionState = "normal" | "hover" | "selected" | "disabled";
```

Comment:
"active" should account for future keyboard navigation as well, but together with "selected" it does introduce some confusion.